zcash_primitives/src/transaction/mod.rs

```diff
@@ -127,9 +143,11 @@ impl TxVersion {
     pub fn version_group_id(&self) -> u32 {
         match self {
             TxVersion::Sprout(_) => 0,
-            TxVersion::Overwinter => OVERWINTER_VERSION_GROUP_ID,
-            TxVersion::Sapling => SAPLING_VERSION_GROUP_ID,
-            TxVersion::Zip225 => V5_VERSION_GROUP_ID,
+            TxVersion::V3 => V3_VERSION_GROUP_ID,
+            TxVersion::V4 => V4_VERSION_GROUP_ID,
+            TxVersion::V5 => V5_VERSION_GROUP_ID,
+            #[cfg(zcash_unstable = "nu7")]
+            TxVersion::Zip230 => V6_VERSION_GROUP_ID,
```

Comment:
```suggestion
            TxVersion::V6 => V6_VERSION_GROUP_ID,
```
This is the cause of the build failures I think.